src/translator.js

```diff
@@ -108,9 +111,7 @@ class Translator {
       if (text) {
         element[property] = text;
       } else {
-        element[property] = element.dataset.i18n;
-
-        console.warn(`Could not find text for attribute "${key}".`);
+        console.error(`Could not find text for attribute "${key}".`);
```

Comment:
`text` should not be empty here, but i'd keep it as error in case something changes in future.